### ISSUE TITLE
`mq-info` procedure call fails  

### ISSUE THREAD

**Issue by narslan** [iraikov/chicken-posix-mq#1]
Hey,
I started writing a CLI,  like this project [mq](https://github.com/goeb/mq)  using `chicken-posix-mq`. 
Since the number of all of the procedures is small, it is not a big task to write an interface or it.

However I stumbled upon a problem while trying to use the procedure `mq-info`  which fails with an error . 
I couldn't understand what the error log points out.
You can reproduce the error if you add (mq-info path) to the test file. 
It fails and produces following error.
```
Error: (cadr) bad argument type: #<invalid forwarded object>
```
This is the test file of the library `run.scm`
```scheme
(import scheme (chicken format) (chicken random) (chicken file posix) (chicken string) (chicken blob) posix-mq)

(print posix-mq?)
(let* ((str "Hello, world!")
       (path (sprintf "/mqtest~A" (pseudo-random-integer 100)))
       (fd (mq-create path  10 1000 oflags: (list open/rdwr)))
       (msg (string->blob str)))
    (mq-info path) ;; -> this makes test fail
    (print "sending " msg)
    (mq-send path msg)
    (print "received " (mq-recv path))
    (mq-unlink path)
  )

```
I digged around in the source file to find the problem,  but I couldn't find a clue. 
So I ended up being here.
Thanks,

**Comment by iraikov** [comment]
Hello, thank you for trying to use chicken-posix-mq and for reporting this issue. I have added a fix for mq-info and created a new release (1.1). Let me know if it works for you.

**Comment by narslan** [comment]
It works very well. I'm very happy with that.
